cob_bringup/robots/cob4-2.xml

```diff
@@ -200,6 +200,25 @@
 		</include-->
 	</group>
 
+	<group>
+		<machine name="cob4-2-s1" address="$(arg cob4-2-s1)" env-loader="$(find cob_bringup)/env.sh" default="true" timeout="30"/>
+		<!-- pc monitor -->
+		<include file="$(find cob_bringup)/tools/pc_monitor.launch" >
```

Comment:
we probably have to update diagnostics configuration to include cob4-2-s1 pc in the PC diagnostics.